webapp/src/js/bootstrapper/index.js

```diff
@@ -112,17 +129,21 @@
       err.status = 401;
       return redirectToLogin(dbInfo, err, callback);
     }
+
     if (hasFullDataAccess(userCtx)) {
       return callback();
     }
+    
+    translator.setLocale(userCtx.locale);
```

Comment:
It doesn't look like `translator` has been used up to this point.  Could it just be initialised with the `locale` here?

Reply:
It is certainly possible. Translator is used in `setUiStatus`, `setUiError`, and `initialReplication`. So initialising it here just means injecting it into a bunch of functions. I don't have a strong feeling either way.